operator/v1/types_console.go

```diff
@@ -22,12 +22,26 @@ type ConsoleSpec struct {
 	// customization options to the web console.
 	// +optional
 	Customization ConsoleCustomization `json:"customization"`
+	// providers contains configuration for using specific service providers.
```

Comment:
+optional, omitempty?

Reply:
We probably could have put on `Customization` as well.